doc/source/dev/roadmap-detailed.rst

```diff
@@ -272,23 +288,16 @@ maintenance.  No major plans or wishes here.
 
 optimize
 ````````
-Overall this module is in good shape. Two good global optimizers were added in
-1.2.0; large-scale optimizers is still a gap that could be filled.  Other
-things that are needed:
-
-- Many ideas for additional functionality (e.g. integer constraints) in
-  ``linprog``, see `gh-9269 <https://github.com/scipy/scipy/issues/9269>`__.
-- Add functionality to the benchmark suite to compare results more easily
-  (e.g. with summary plots).
-- deprecate the ``fmin_*`` functions in the documentation, ``minimize`` is
-  preferred.
-- ``scipy.optimize`` has an extensive set of benchmarks for accuracy and speed of
-  the global optimizers. That has allowed adding new optimizers (``shgo`` and
-  ``dual_annealing``) with significantly better performance than the existing
-  ones.  The ``optimize`` benchmark system itself is slow and hard to use
-  however; we need to make it faster and make it easier to compare performance of
-  optimizers via plotting performance profiles.
+It is very challenging to write and maintain optimizers that approach the
+state of the art for large-scale problems. In many cases, it will be more
+appropriate for SciPy to add interfaces to carefully-vetted,
+externally-maintained projects (e.g. HiGHS, PRIMA) rather than vendor
+existing optimizers or introduce custom implementations. Other areas
+for future work include the following.
 
+- Improve the interfaces of existing optimizers (e.g. ``shgo``).
```

Comment:
```suggestion
- Improve the interfaces of existing optimizers (e.g. ``shgo``).
```
I single out SHGO mostly to give an idea of what I mean - I think many of the [issues](https://github.com/scipy/scipy/issues?q=is%3Aissue+is%3Aopen+shgo+label%3Ascipy.optimize) have to do with a mismatch between the documented options and what the code does. Other functions have a ton of options, and I'm not sure that the interfaces are as consistent as they could be or that all the options are well documented/tested.